tests/functional/adapter/test_timestamps.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from dbt.tests.adapter.utils import test_timestamps
+
+_MODEL_CURRENT_TIMESTAMP = """
+SELECT {{current_timestamp()}} as current_timestamp,
+       {{current_timestamp_in_utc_backcompat()}} as current_timestamp_in_utc_backcompat,
+       {{current_timestamp_backcompat()}} as current_timestamp_backcompat
+"""
+
+
+class TestCurrentTimestampSnowflake(test_timestamps.BaseCurrentTimestamps):
```

Comment:
Per @gshank's comment [here](https://github.com/dbt-labs/dbt-core/pull/5838#discussion_r984598814), let's make this consistent with the rest of the code base.
```suggestion
from dbt.tests.adapter.utils.test_timestamps import BaseCurrentTimestamps

_MODEL_CURRENT_TIMESTAMP = """
SELECT {{current_timestamp()}} as current_timestamp,
       {{current_timestamp_in_utc_backcompat()}} as current_timestamp_in_utc_backcompat,
       {{current_timestamp_backcompat()}} as current_timestamp_backcompat
"""


class TestCurrentTimestampSnowflake(BaseCurrentTimestamps):
```